modules/ROOT/pages/to-configure-api-settings.adoc

```diff
@@ -4,28 +4,24 @@ include::_attributes.adoc[]
 endif::[]
 :imagesdir: ../assets/images/
 
-Anypoint Exchange lets you configure whether REST, SOAP, and HTTP API services are public or private in a Private Exchange or in an Exchange Portal. Typical services that you would set visibility for are proxies or policies associated with your API that you set using API Manager.
+Use Anypoint Exchange to configure whether REST, SOAP, and HTTP API services are public or private in a private Exchange or in an Exchange portal. For example, configure public or private visibility for proxies or policies associated with your API and set using API Manager.
 
-After you register an API from Exchange in API Manager, the instances for the API appear in Exchange in the API Instances list in the left navigation bar.
+After you register an API from Exchange in API Manager, the instances for the API appear in Exchange in the API instances list in the navigation bar.
 
-You can also create an API instance if your API is managed by another API management program outside of Anypoint Platform.
+Exchange also creates API instances for APIs managed by another API management program outside of Anypoint Platform.
 
-Access differs by Exchange type:
+Access differs by Exchange type. In a private Exchange, the API instances list appears for HTTP, REST, and SOAP APIs. In an Exchange portal, the API instances list appears for SOAP and HTTP APIs. All instances in an Exchange portal are read only.
```

Comment:
> Access differs by Exchange type. In a private Exchange, the API instances list appears for HTTP, REST, and SOAP APIs. In an Exchange portal, the API instances list appears for SOAP and HTTP APIs

I'm not sure when did we add this, but it's not accurate. The API instances list appear for HTTP, REST and SOAP APIs in both private Exchange and Exchange Public Portals (also I assume that "Exchange portal" refers to "Exchange Public Portal" in this context, let's make it more explicit by using the latter term). I don't think there is a need to call this out.

>All instances in an Exchange portal are read only.

This is accurate 👍  (assuming we replace "Exchange portal" with "Exchange public portal"). This phrase could be moved after the following paragraph that discusses API Instance visibility.

Reply:
Thank you! Fixed.